### PR TITLE
WT-9438 Skip cppsuite files for spell check

### PR DESCRIPTION
**Pull request by etienneptl**
Make `s_script` skip all the files in the `test/cppsuite` folder. This will unblock WT-9385 and is a temporary change that will be fixed by WT-9433.

### CODE REVIEW

File: dist/s_string [hunk 1]
```diff
@@ -45,9 +45,12 @@ check() {
 }
 
 # List of files to spellchk.
+# FIXME-WT-9433 Skip files in the test/cppsuite folder.
 l=`(cd .. &&
-    find bench examples ext src test -name '*.[chsy]' &&
-    find src -name '*.in' && find test -name '*.cxx' && find test -name '*.cpp')`
+    find bench examples ext src test -not -path "test/cppsuite/*" -name '*.[chsy]' &&
```
Comment (ajmorton) [inline dist/s_string:50]: Can you also add `-not -path "ext/storage_sources/s3_store/*"`? 
That way it’s explicit we also need to check the s3 code when fixing WT-9433

File: dist/s_string [hunk 2]
```diff
@@ -45,9 +45,12 @@ check() {
...
+    find src -name '*.in' &&
+    find test -name '*.cxx' &&
```
Comment (korteland) [inline dist/s_string:52]: I know this bit is just reformatted from the existing code, but I don't think we have any files matching this criteria any more after we moved everything to be `.cpp`, right? At least, `find` didn't get anything on a clean checkout.

**Review by korteland — APPROVED** [review]
LGTM, one line is maybe unnecessary these days.

File: dist/s_string (hunk 2 above)
Comment (ajmorton) [reply dist/s_string:52]: Agreed. Our only remaining .cxx files are 
```
./bench/workgen/workgen.cxx
./tools/xray_to_optrack/xray_to_optrack.cxx
```
neither of which are covered by the existing find statements.

**Review by ajmorton — APPROVED** [review]
LGTM

File: dist/s_string (hunk 2 above)
Comment (etienneptl) [reply dist/s_string]: Fixed in [b4f8dfd](https://github.com/wiredtiger/wiredtiger/pull/8010/commits/b4f8dfd3b038235ada351b551823c66ca180a223)

**Pull request merged**